web/index.html

```diff
@@ -17,12 +17,12 @@
 		<div id="wrapper">
 			<div id="content">
 				<h1>Vircadia Community Applications Repository</h1>
-				<p>This repository stores optional <b>tablet applications</b> and <b>functionalities</b> provided by the contributions of the community members. 
-				If you have created an Application or functionality script and want to offer it to the community, this is where you can do it.</p>
+				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members who are not core developers of Vircadia. 
```

Comment:
```suggestion
				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members. 
```
Core developer or not, doesn't play any role here. For example, Davids radar app is in here as well.